draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1709,6 +1712,13 @@ data. The original JSON data is then used by the application. See
 
    [[ To be removed from the final specification ]]
 
+   -08
+
+* Make RFCs 0020 and 7515 normative references
+* Be a bit more prescriptive in suggesting RFC7800 cnf/jwk be used to convey the Key Binding key
+* Editorial changes aimed at improved clarity
```

Comment:
This looks like it needs more indentation.

Reply:
The resultant output is okay https://drafts.oauth.net/oauth-selective-disclosure-jwt/yasskin-asking/draft-ietf-oauth-selective-disclosure-jwt.html#name-document-history

Reply:
I didn't expect that, but in that case this is not an issue, thanks!